tests/integration/store/test_store_register.py

```diff
@@ -32,6 +32,11 @@ def test_successful_private_registration(self):
         snap_name = self.get_unique_name()
         self.register(snap_name, private=True)
 
+    def test_successful_registration_with_store(self):
+        self.login()
+        snap_name = self.get_unique_name(name_prefix="snapcraft-test")
```

Comment:
What's the reason for using a different name prefix here ? it implies in additional cleanup steps on our staging service for no particular gain. We can continue to differentiate the name (in case it falls into the global store) by extending the agreed prefix, e.g.`test-snapcraft-in-store`.

Reply:
I only did that because brand stores now require the prefix for the registered names to be that of the brand store.

Reply:
Ah, forgot about the store name prefix, which works both ways (also blocking the use in the global store).Any objections to set the prefix to `test-snapcraft-in-store` ?

Reply:
I've discussed with Adam and we updated the store name prefix to `test-snapcraft-in-store` to simplify the cleanup.